example-react-app/src/app/AppMenu.tsx

```diff
@@ -18,265 +18,265 @@ export const AppMenu = (props: AppMenuProps) => {
         screenId={"ExampleCustomScreen"}
         icon={<BarsOutlined />}
         caption={"screen.ExampleCustomScreen"}
-        key={"81b693e6-2a21-48df-8313-2b352df20497"}
+        key={"f99a26c3-089f-4292-afdb-387f1dfb39d9"}
```

Comment:
Please revert changes to AppMenu unless you are adding/removing menu items. Keys are randomly generated during bootstrap-react-app, committing them will just create unnecessary conflicts.

Ideally, even when adding/removing menu items only commit the added/removed item and revert regenerated keys.